docs/reference/commandline/build.md

```diff
@@ -96,22 +96,22 @@ For example, run this command to use a directory called `docker` in the branch
 `container`:
 
 ```bash
-$ docker build https://github.com/docker/rootfs.git#container:docker
+$ docker build https://github.com/docker/rootfs#container:docker
```

Comment:
Sorry, it was my bad, the change is incorrect.
Git is installed on the machine, but I have tried to run commands in Windows PowerShell ISE (my work machine). On MAC (home machine) all works like a charm and on my work machine all works in cmd.